elastic/kibana
# [Fleet] fixing issue of returning output api key

## Summary

Closes https://github.com/elastic/ingest-dev/issues/1944
Added mapping of known fields instead of returning all fields from `.fleet-agents` in agents API.

To verify:
- create an agent policy with a new output
- enroll an agent to it
- call agents API and verify that the outputs section doesn't have api key

```
GET kbn:/api/fleet/agents
{
  "items": [
    {
      "id": "504b3006-52df-46a6-b7db-f3dc67aca7ac",
      "type": "PERMANENT",
      "active": true,
      "enrolled_at": "2023-06-07T07:45:30Z",
     ...
      "outputs": {
        "68233290-0486-11ee-97a3-c3856dd800f7": {
          "api_key_id": "En_RlIgBn_WkCEINb-pQ",
          "type": "elasticsearch"
        },
        "default": {
          "api_key_id": "FH_RlIgBn_WkCEINcOrY",
          "type": "elasticsearch"
        }
      },
```

### Checklist
